packages/vue3-vite/src/util/defaults.ts

```diff
@@ -34,17 +35,21 @@ export const CypressDevDependencies = {
   '@nrwl/cypress': '*',
 };
 
+export const VueDevDependencies = {
+  '@vue/test-utils': '^2.2.6',
+};
+
 export const VitestDevDependencies = {
   '@nrwl/vite': '^15.4.4',
-  '@vue/test-utils': '^2.2.6',
   'happy-dom': '^8.1.1',
   vitest: '^0.26.3',
 };
 
 export const JestDevDependencies = {
-  '@babel/preset-env': '^7.20.2',
+  '@nrwl/jest': nrwlJestVersion,
```

Comment:
I separate Jest, Vitest, and Vue dependencies into different objects.
It would be great if you could double check if I'm doing it correctly. 😬

Reply:
Looks good to me!